packages/create-miniapp-cli/template/server/jest.setupTest.js

```
@@ -0,0 +1,22 @@
+const index = require('@app/~/index')
```

ESLint

[error] '@app/~/index' module is restricted from being used by a pattern.

(no-restricted-modules)